index.html

```diff
@@ -4,9 +4,95 @@
+                    <p class="card-descrep">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusamus non dolorem excepturi libero .</p>
+                </div>
+                <div class="card-work">              
+                    <div class="card-icon"><i class="fa fa-users" aria-hidden="true"></i></div>
+                    <h3 class="card-head">Expert Teachers</h3>
+                    <p class="card-descrep">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusamus non dolorem excepturi libero .</p>
+                </div>
+                <div class="card-work">             
+                    <div class="card-icon"><i class="fa fa-laptop" aria-hidden="true"></i></div>
+                    <h3 class="card-head">Online Learning</h3>
+                    <p class="card-descrep">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusamus non dolorem excepturi libero .</p>
+                </div>
+                <div class="card-work">                    
+                    <div class="card-icon"><i class="fa fa-microphone" aria-hidden="true"></i></div>
+                    <h3 class="card-head">Audio Lessons</h3>
+                    <p class="card-descrep">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusamus non dolorem excepturi libero .</p>
+                </div>
+                <div class="card-work">          
+                    <div class="card-icon"><i class="fa fa-film " aria-hidden="true"></i></div>
+                    <h3 class="card-head">Video Lessons</h3>
+                    <p class="card-descrep">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusamus non dolorem excepturi libero .</p>
+                </div>
+            </section>
+            <div class="join">
+                <button type="button">
+                    See Our Tutorials 
+                </button>
+            </div>
+        </section>
+        <section class="instructors">
+            <div class="title">
+                <h2>Our instructors</h2>
+                <hr class="line-break">
+            </div>
+        </section>
+        <section class="partners">
+            <h2>Partners</h2>
+            <hr class="line-break">
+            <div class="companies">
+                <div class="partner"></div> 
+                <div class="partner"></div>
+                <div class="partner"></div>
+                <div class="partner"></div>
+                <div class="partner"></div>
+            </div>
+        </section>
+        <footer>
+            <div class = 'copy-logo'><img src="./images/Logo.svg" alt=" copy right Logo"></div>
+            <div class="copy-right">
+                <p> 2021 Ahmed Dev Egypt. All Rights Reserved</p>
+                <p>Ahmed Dev company, is a non-profit company, for any purpose related to this, please contact ahmdadl563@gmail.com</p>
+            </div>
+        </footer>
 </body>
 </html>
```

Comment:
- [ ] No functionality has been added to your project. Please add proper functionalities to your project.